webapp/ordering/models.py

```diff
@@ -234,6 +240,33 @@ def send_reminder_mails(self):
                 mail_template, user=user, order_round=self)
             mail_user(user, *rendered_template_vars)
 
+    def get_rides(self):
+        from transport.models import Ride
```

Comment:
Are you sure this method is needed? You should be able to reference `self.ride.all()` (the `related_name` on `order_round` FK on `Ride` is called `ride` (ideally would be `rides` :))

Reply:
You're absolutely right, that's way easier.